nginx-autoinstall.sh

```diff
@@ -235,6 +241,28 @@ case $OPTION in
 
 			./config
 		fi
+		
+		# ModSecurity
+		if [[ "$MODSEC" = 'y' ]]; then
+			cd /usr/local/src/nginx/modules || exit 1
+			git clone --depth 1 -b v3/master --single-branch https://github.com/SpiderLabs/ModSecurity
+			cd ModSecurity
+			git submodule init
+			git submodule update
+			./build.sh
+			./configure
+			make
+			make install
+			mkdir /etc/nginx/modsec
+			wget -P /etc/nginx/modsec/ https://raw.githubusercontent.com/SpiderLabs/ModSecurity/v3/master/modsecurity.conf-recommended
+			mv /etc/nginx/modsec/modsecurity.conf-recommended /etc/nginx/modsec/modsecurity.conf
+
+			read -p "  Enable nginx ModSecurity? [y/n]: " -e MODSEC_ENABLE
```

Comment:
Maybe this should be asked at the beginning right after `$MODSEC` and if `$MODSEC` is true? We should gather all the user input at the same step.

Reply:
Totally agree. Moved as requested. :+1: